bazel/dep/deps_extra.bzl

```diff
@@ -0,0 +1,8 @@
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
+load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
+
+# Wasm deps that rely on a first stage of dependency loading in wasm_dependencies().
+def wasm_dependencies_extra():
+    bazel_skylib_workspace()
+    rules_proto_dependencies()
+    rules_proto_toolchains()
```

Comment:
Why is this split into 2 separate files and 2 separate targets?

Reply:
Because bazel's requirement that import has to be on the top of bzl file (there is a comment for this at L4). This follows envoy's pattern of two steps importing: https://github.com/envoyproxy/envoy/blob/935ded9b2d5c0279bdefb984d3e2e6086be98ea9/WORKSPACE#L10-L17